src/current/cockroachcloud/cloud-api.md

```diff
@@ -154,12 +163,15 @@ curl --request POST \
 {
   "name": "notorious-moose",
   "provider": "GCP",
+  "plan": "STANDARD",
```

Comment:
You don't need this entire example if you take my suggestion to remove the `raw` sections and thus remove the filter-tabs.